### PR TITLE
feat: Add new check to code-checker

### LINKED ISSUE #10454: Add new check to code-checker
Everything looks good in code-checker with this artist but anyways there is still some issue in the code is not covered yet on the code-checker page.

Code from the artist: 
[xest_2024_06_08_15_02_54 (1).zip](https://github.com/user-attachments/files/15787675/xest_2024_06_08_15_02_54.1.zip)

We are unable to open the canvas page. If we use this code in code-checker and the user clicks "open in the new tab" from our gallery detail page, it will show a blank page. Error in the console:

Screenshot 2024-06-11 at 17 59 27

- https://playground-r2.koda.art/codeChecker/348371ba-4461-4afd-81a2-f7f8447f708a.html?hash=0xeb00330d03a53748d2f2b010086752684e3a34b7ec97aeabe208d0986524be09

### We need this issue to be spotted in the code-checker

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #10454

## Needs QA check

- @kodadot/qa-guild please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-06-18 at 14 43 28@2x

image

### CODE REVIEW

**Review by vikiival — COMMENTED** [review]
please consult kodahash :)

File: components/codeChecker/issueHint/NotUsingExternalResources.vue [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <CodeCheckerIssueHintCodeBlock class="mt-5" :code="code" lang="html" />
+</template>
+
+<script setup lang="ts">
+const code = `
+  <!-- DON'T ❌ -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.9.2/p5.js" />
+
+  <!-- DO ✅-->
+  <script src="p5.js" />
```
Comment (vikiival) [inline components/codeChecker/issueHint/NotUsingExternalResources.vue:15]: [incorrect](https://github.com/vikiival/kodahash/blob/main/public/index.html)

```suggestion
  <script src="./p5.min.js">
```

**Review by vikiival — CHANGES_REQUESTED** [review]

File: components/codeChecker/issueHint/NotUsingExternalResources.vue (hunk 1 above)
Comment (hassnian) [reply components/codeChecker/issueHint/NotUsingExternalResources.vue]: fixed

File: components/codeChecker/issueHint/NotUsingExternalResources.vue [hunk 2]
```diff
@@ -12,6 +12,6 @@ const code = `
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.9.2/p5.js" />
```
Comment (vikiival) [inline components/codeChecker/issueHint/NotUsingExternalResources.vue:12]:
```suggestion
  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.9.2/p5.min.js" />
```

**Comment by vikiival** [comment]
cc @Jarsen136

**Review by vikiival — APPROVED** [review]

**Review by Jarsen136 — APPROVED** [review]

**Pull request merged**